tests/_client.lua

```diff
@@ -1,5 +1,10 @@
 local spawn = require("awful.spawn")
 
+local lua_executable = os.getenv("LUA_EXECUTABLE")
```

Comment:
Can we change the envar name ? In other Linux `make` oriented programs, the envar is the same as the program's name capitalized, so this should be LUA

Reply:
I guess that's not on me to decide. The CMake files already contained a few `X_EXECUTABLE` variables, and I figured using the same name for the envvar would help keep the connection obvious in the long run.

But it can certainly be changed.

Reply:
I haven't enough experience with cmake. Is it possible to keep the `X_EXECUTABLE` variable for cmake and only change the name of the exposed environment variable?

I mean, in the `awesomeConfig.cmake` file, use `LUA=${}` instead of `LUA_EXECUTABLE=${}`:

```cmake
${CMAKE_COMMAND} -E env CMAKE_BINARY_DIR='${CMAKE_BINARY_DIR}' LUA='${LUA_EXECUTABLE}' ${TESTS_RUN_ENV} ./tests/themes/run.sh
```

Reply:
Yes, that would be possible.

Reply:
I'm guessing (didn't check it myself but at least it is common it Linux to be so) that you can export LUA variable on the command line. Something like this:

`LUA=/usr/bin/lua5.3 make`

(just a side note, everyone can specify it as he/she wishes)